### ISSUE TITLE
Failure during "Install necessities and nice-to-haves"

### ISSUE THREAD

**Issue by dutts** [sovereign/sovereign#793]
```
TASK [common : Install necessities and nice-to-haves] **************************
[DEPRECATION WARNING]: Invoking "apt" only once while using a loop via 
squash_actions is deprecated. Instead of using a loop to supply multiple items 
and specifying `pkg: "{{ item }}"`, please use `pkg: ['apache2', 'apt-
transport-https', 'apticron', 'build-essential', 'debian-goodies', 'git', 
'htop', 'iftop', 'iotop', 'molly-guard', 'mosh', 'python3-software-properties',
 'ruby', 'screen', 'sudo', 'unattended-upgrades', 'vim', 'zsh']` and remove the
 loop. This feature will be removed in version 2.11. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
failed: [178.79.186.238] (item=[u'apache2', u'apt-transport-https', u'apticron', u'build-essential', u'debian-goodies', u'git', u'htop', u'iftop', u'iotop', u'molly-guard', u'mosh', u'python3-software-properties', u'ruby', u'screen', u'sudo', u'unattended-upgrades', u'vim', u'zsh']) => {"ansible_loop_var": "item", "changed": false, "item": ["apache2", "apt-transport-https", "apticron", "build-essential", "debian-goodies", "git", "htop", "iftop", "iotop", "molly-guard", "mosh", "python3-software-properties", "ruby", "screen", "sudo", "unattended-upgrades", "vim", "zsh"], "msg": "value of state must be one of: absent, build-dep, fixed, latest, present, got: installed"}
```
I'm getting the above running the ansible script.

**Comment by neuhaus** [comment]
What Linux distribution (with version) are you running it on?
What version if Ansible are you using?

**Comment by codecowboy** [comment]
This could be related to the Ansible version. I just logged a separate issue when using vagrant. What ansible version are you running locally?

**Comment by ruanha** [comment]
getting the same error, Ansible version 2.9.4
